extension/src/test/suite/vscode/quickPick.test.ts

```diff
@@ -208,8 +208,16 @@ suite('Quick Pick Test Suite', () => {
         { title: 'select up to 3 values' as Title },
         maxSelectedItems
       )
+      const selectionEvent = new Promise(resolve =>
+        quickPick.onDidChangeSelection(items => {
+          if (items.length === 3) {
```

Comment:
Going by this error:

```
    2) Quick Pick Test Suite
         quickPickUserOrderedValues
           should limit the number of values that can be selected to the max selected items:
  
        the max number of items are selected
        + expected - actual
  
        -2
        +3
        
    	at d:\a\vscode-dvc\vscode-dvc\extension\dist\test\suite\vscode\quickPick.test.js:147:105
    	at Generator.next (<anonymous>)
    	at fulfilled (d:\a\vscode-dvc\vscode-dvc\extension\dist\test\suite\vscode\quickPick.test.js:5:58)
```

`await executeCommand` could possibly not be resolving when the command is 100% finished. So I've tried to add an extra await event that checks for when there are three selected items.

Reply:
Or maybe the quick pick select commands aren't being run or being run correctly for some reason 🤔